src/raytracer/LOSOutputGenerator.m

```diff
@@ -117,7 +117,7 @@
             output1(16:17) = PolarizationTx(2,:);
         end
     end
-    output1(18) = 0;
+    output1(18) = mod(distance/lambda*2*pi,2*pi);
```

Comment:
this is an important change with respect to what was done before. external simulator might (and do!) rely on this term to be computed offline. why has it been added? is it necessary? do people relying on this agree and expect this change?